packages/components/psammead-script-switch/src/index.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { dirDecorator } from '@bbc/psammead-storybook-helpers';
+import styled from 'styled-components';
+import notes from '../README.md';
+import ScriptSwitch from './index';
+
+const Container = styled.div`
+  background-color: black;
```

Comment:
could this be updated to the red used in the navigation? `#b80000`

Reply:
@jblaut I thought about that but looking at [BrandSvgs](https://bbc.github.io/psammead/?path=/story/utilities-svgs-brand-svgs--afaanoromoo), even though we know that they will mostly be used on the brand, we are not applying ` #b80000` as the container background. I think we are applying `black` to avoid confusion on what `bg-color` is part of the component.

Reply:
ahh fair enough. Then just 👍 from me 😄